tests/cpptests/test_cpp_iterator_intersection.cpp

```diff
@@ -194,7 +194,7 @@ class IntersectionIteratorTest : public ::testing::Test {
     for (auto &term : terms) {
       if (invertedIndexes.find(term) == invertedIndexes.end()) {
         // Create a new inverted index for the term if it doesn't exist
-        invertedIndexes[term] = NewInvertedIndex((IndexFlags)(INDEX_DEFAULT_FLAGS), &dummy);
+        invertedIndexes[term] = NewInvertedIndex((IndexFlags)(INDEX_DEFAULT_FLAGS | Index_WideSchema), &dummy);
```

Comment:
Testing `wide` by default now?

Reply:
The test is already making entries with the wide flag on line 208 because `RS_FIELDMASK_ALL` is used. The Rust code needs the written entry and the encoder to align at runtime hence why the flags for the tests were updated in this commit https://github.com/RediSearch/RediSearch/pull/6851/commits/005dc54fad1163fd366a3b6238c298cd579bc8cd